qtglean/src/lib.rs

```diff
@@ -57,6 +57,11 @@ pub extern "C" fn glean_initialize(is_telemetry_enabled: bool, data_path: FfiStr
     };
 
     register_pings();
+    if channel == "staging" {
+        glean::set_debug_view_tag("MozillaVPNRust");
+        glean::set_log_pings(true);
+    };
```

Comment:
I am not pro adding this. These can be set using environment variables when necessary. There is also a ticket to add these options to the dev menu in the future. Setting the debug view tag for staging at all times will spam the debug viewer which is bad, we def should _not_  do it. Setting the log pings spams the VPN logs, which I might be persuaded to agree with though. WDYT?

Reply:
I 100% agree on the debug view tag; it's been removed.

I also agree that logging pings makes the logs dirtier, but our staging logs are somewhat busy already, and for testing that doesn't necessarily seem "wrong" to me; I end up using a lot of filtering and searching. I don't feel super strongly about this, though - just say the word and I'll pull it out.

Reply:
I'd prefer we get into the habit of turning on log pings with environment variables / add the option to the dev menu. The logs are already verbose, true. However I don't see that as a good reason to add stuff to them when it's not necessary.

Reply:
Fair enough. I removed it. Thanks!